src/main/java/digital/slovensko/autogram/ui/UI.java

```diff
@@ -48,5 +48,9 @@ public interface UI {
 
     void onPDFAComplianceCheckFailed(SigningJob job);
 
+    public void onSignatureValidationCompleted(SigningJob job);
+
+    public void onSignatureCheckCompleted(SigningJob job);
+
```

Comment:
Tu radsej poslime ten report ako separe parameter ako to posuvat cez signing job

Reply:
@jsuchal Podľa signingJobu sa potom nájde príslušný controller, takže asi to nepôjde bez toho.

Reply:
Mozeme to spravit naopak. Report bude obsahovat odkaz na signingjob a uz vies.

Reply:
Report je ale objekt z DSS. Chceme nad tým vytvoriť ďalší wrapper?

Reply:
Vyrobil som core.ValidationReportsWrapper